crates/astria-sequencer/src/authority/action.rs

```diff
@@ -23,6 +24,7 @@ impl ActionHandler for tendermint::validator::Update {
         &self,
         state: &S,
         from: Address,
+        _fee_asset_id: &asset::Id,
```

Comment:
Oh, so all of these APIs are not special casing native asset. I think that's even more of a reason to not special case this in the protof :)